dbt/models/sec10k/core_sec10k__assn_exhibit_21_subsidiaries_and_eia_utilities/schema.yml

```diff
@@ -5,10 +5,6 @@ sources:
       - name: core_sec10k__assn_exhibit_21_subsidiaries_and_eia_utilities
         data_tests:
           - expect_columns_not_all_null
-          - check_row_counts_per_partition:
-              table_name: core_sec10k__assn_exhibit_21_subsidiaries_and_eia_utilities
-              partition_expr: null
-              severity: warn
```

Comment:
The row counts for this table vary by +/- 1 between the local full ETL and the nightly builds. We don't know why. It was set to warn rather than fail, which avoids dbt test failures resulting from this discrepancy, but updating all row counts locally vs. in the nightly builds will result in changes to the row count file, which will result in changes in the checked out repo in the nightly builds that will block updates to the `nightly` or other tags.